spotinst_sdk2/clients/ocean/__init__.py

```diff
@@ -3,20 +3,24 @@
 from spotinst_sdk2.client import Client
 import spotinst_sdk2.models.ocean.aws as aws_ocean
 
+
 class OceanAwsClient(Client):
     __base_ocean_url = "/ocean/aws/k8s/cluster"
+    __base_ocean_url1 = "/ocean/aws/k8s/launchSpec"
```

Comment:
Change to __base_ocean_cluster_url and __base_ocean_launchspec_url

Reply:
Done